recipes/links/build.sh

```diff
@@ -5,13 +5,22 @@ cp  LINKS $PREFIX/bin
 chmod +x $PREFIX/bin/LINKS
 
 
+set -eux -o pipefail
+
 #Recompiling C code
 cd lib/bloomfilter/swig/
 
+if [[ $OSTYPE == linux* ]]; then
+	suffix="so"
+else
+	suffix="bundle"
```

Comment:
Don't you want .dylib here?

Reply:
I had `bundle` there before just because it was what is in the `brew` recipe, but I changed to `.dylib` and that also seems to work just fine!